si2.dal/Context/Si2DbContext.cs

```diff
@@ -42,6 +44,12 @@ protected override void OnModelCreating(ModelBuilder builder)
             base.OnModelCreating(builder);
 
 			builder.Entity<BookCategory>().HasKey(bc => new { bc.BookId, bc.CategoryId });
+
+
+			builder.Entity<ProgramLevel>().HasIndex(pl => pl.nameFr).IsUnique();
```

Comment:
Check if we need to rname these idx so that we control what is the name given in the SQL db - check with Tania